manifests/params.pp

```diff
@@ -21,4 +23,27 @@
       fail("${::operatingsystem} not supported")
     }
   }
+
+  case $::operatingsystemmajrelease {
+    7, 8: {
```

Comment:
Im not sure what version of ubuntu moved to systemd but that should be here too. This shoulld work for rhel/centos7 and debian 7 and 8.